pyproject.toml

```diff
@@ -1,10 +1,10 @@
 [build-system]
-requires = ["poetry-core"]
+requires = ["poetry>=1.4.0"]
```

Comment:
This was changed to `poetry-core` in #108. Do you know if one or the other is better?

Reply:
I think `poetry-core` is actually more proper. The intent here was just that was had a modern version of poetry (at least 1.2.0).  I'll revert this (and apply it to some of my other projects 😄 )